src/main/scala/org/apache/sysml/api/ml/BaseSystemMLClassifier.scala

```diff
@@ -225,7 +226,7 @@ trait BaseSystemMLClassifierModel extends BaseSystemMLEstimatorModel {
     // ml.setExplainLevel(ExplainLevel.RECOMPILE_RUNTIME)
     val modelPredict = ml.execute(script._1.in(script._2, X, new MatrixMetadata(X.getNumRows, X.getNumColumns, X.getNonZeros)))
     val ret = PredictionUtils.computePredictedClassLabelsFromProbability(modelPredict, isSingleNode, sc, probVar)
-              .getBinaryBlockMatrix("Prediction").getMatrixBlock
+              .getMatrix("Prediction").toMatrixBlock
```

Comment:
down the road, it's probably a good idea to modify the APIs of these pipelines to return the model as Matrix instead of as MatrixBlock in order to avoid OOMs in case of huge models.